external/js/kyber/webpack.config.js

```diff
@@ -39,18 +12,30 @@ const browserConfig = {
     rules: [
       {
         test: /\.js$/,
-        exclude: /(node_modules|bower_components)/,
+        exclude: /node_modules/,
         use: {
           loader: "babel-loader",
           options: {
-            presets: ["env"],
-	    plugins: [ require("babel-plugin-transform-object-rest-spread") ],
+            presets: ["env"]
           }
         }
+      },
+      {
+        test: /\.tsx?$/,
```

Comment:
I wondered (and lost some time last time I did TS) about this .tsx extension. Can we remove it? Seems like some windows thing. So here, change to /\.ts$/ and below remove it.